eZ/Publish/API/Repository/ContentService.php

```diff
@@ -432,4 +432,15 @@ public function newTranslationInfo();
      * @return \eZ\Publish\API\Repository\Values\Content\TranslationValues
      */
     public function newTranslationValues();
+
+    /**
+     * Returns true if the given content type $contentType has content instances.
+     *
+     * @since 6.0.1
+     *
+     * @param \eZ\Publish\API\Repository\Values\ContentType\ContentType $contentType
+     *
+     * @return bool
+     */
+    public function isContentTypeUsed(ContentType $contentType);
```

Comment:
@bdunogier @crevillo @andrerom Let's see... some options:
1. Move everything to ContentType
2. Move API to ContentType, keep SPI in Content (since it's content related)
3. Keep everything in Content, optionally renaming methods to show it's content related)

@crevillo Good point about ContentType being used in policies, I may have to add that check.